src/helpers/email/send.ts

```diff
@@ -71,26 +71,28 @@ export async function newRequest(requestObj: any): Promise<any> {
 			if (!manager) {
 				return false;
 			} else if (manager.emails) {
-				let msg = requestTemplate(
-					requestObj.quantity,
-					requestingUser.name,
-					manager.name,
-					item.name,
-					store.name,
-					store._id,
-				);
-				let email = {
-					from: `ETHS Kitcoin Team <${email_from}>`,
-					replyTo: email_replyto,
-					to: manager.email,
-					subject:
-						`${requestingUser.name} requested ` +
-						(requestObj.quantity > 1 ? 'items!' : 'an item!'),
-					html: msg,
-				};
-				await transporter.sendMail(email).catch(console.log);
+				emails.push(manager.email!);
```

Comment:
`User.email` is optional. Why treat it like it's always set? Nodemailer doesn't expect an `undefined` value in the email list either.

Reply:
How would User.email ever be undefined? The only way to sign into Kitcoin is with an Eths202.org Google account, which should (always?) have an email. Should we set it to required?